inst/components/sidebar.scss

```diff
@@ -1,8 +1,11 @@
 $bslib-sidebar-padding: $spacer * 1.5 !default;
 $bslib-sidebar-icon-size: $spacer !default;
-
+$bslib-sidebar-bg: var(--bslib-sidebar-bg, #{$gray-200}) !default;
 $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid var(--bs-card-border-color, #{$card-border-color}) !default;
 
+$bslib-sidebar-column-sidebar: Min(calc(100% - #{$bslib-sidebar-icon-size}), var(--bslib-sidebar-width, 250px));
+$bslib-sidebar-column-main: minmax(0, 1fr);
+
 .bslib-sidebar-layout {
```

Comment:
If we do move forward with the gray-200 (or whatever), let's make sure it looks good with `actionButton()` (and other shiny inputs). Maybe we can set a CSS variables to tweak some styles if need be

Reply:
I went with `$gray-100` for the default, which works nicely out of the box for action buttons and other elements